src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -5094,7 +5132,7 @@ interface I
 
 class C : I
 {
-    public void Goo([DefaultParameterValue(1), Optional] int x = 1, int[,] y = null)
+    public void Goo([{|CS1745:DefaultParameterValue|}(1), {|CS1745:Optional|}] int x = {|CS8017:1|}, int[,] y = null)
```

Comment:
🐛 this code generation is not correct. I'm guess the result we want is:

```suggestion
    public void Goo(int x = 1, int[,] y = null)
```

Reply:
➡️ https://github.com/dotnet/roslyn/issues/54326